### PR TITLE
lslocks: add page

### PR DESCRIPTION
**Pull request by RH-sdavey**
Part of #2214 

- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/lslocks.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+> List all local system locks.
+
+- List all local system locks:
+
+`lslocks`
+
+- List locks with defined column headers:
+
+`lslocks -o PID,COMMAND,PATH`
```
Comment (mebeim) [inline pages/linux/lslocks.md]:
```suggestion
`lslocks --output PID,COMMAND,PATH`
```
Long option names are to be preferred over short ones.

File: pages/linux/lslocks.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- List locks raw output (no columns), and no column headers:
+
+`lslocks -r -n`
```
Comment (mebeim) [inline pages/linux/lslocks.md]:
```suggestion
`lslocks --raw --noheadings`
```
Long option names are to be preferred over short ones.

File: pages/linux/lslocks.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Populate a csv file with the information from above command:
```
Comment (mebeim) [inline pages/linux/lslocks.md]: I don't think this example is needed. After all it's just the same as the previous one with an added pipe which only replaces spaces with commas (also, a CSV doesn't realy *need* to be comma-delimited, CSVs can be space-delimited too). It does not really show different features of the `lslocks` command.

I would suggest adding examples for the `--pid` and `--json` options instead, which are not mentioned. If you don't feel like or don't have the time to add examples for those too, that's fine they can be added in the future, but I wouldn't populate the page with more complex examples without having all the basic functionality covered first.

File: pages/linux/lslocks.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# lslocks
+
+> List all local system locks.
```
Comment (mebeim) [inline pages/linux/lslocks.md]:
```suggestion
> List local system locks.
```
I'd suggest removing "all" here, the command has more functionalities.

File: pages/linux/lslocks.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+- List all local system locks:
+
+`lslocks`
+
+- List locks with defined column headers:
+
+`lslocks -o PID,COMMAND,PATH`
+
+- List locks raw output (no columns), and no column headers:
```
Comment (mebeim) [inline pages/linux/lslocks.md]:
```suggestion
- List locks producing a raw output (no columns), and without column headers:
```
Looks clearer, don't you think?

Comment (RH-sdavey) [reply pages/linux/lslocks.md]: Yes, long friday and brain wasn't working 👍

File: pages/linux/lslocks.md (hunk 3 above)
Comment (RH-sdavey) [reply pages/linux/lslocks.md]: Ok, will remove this example ( I only added that as I used it myself quite recently ) and add the `--pid` and `--json` examples after work this evening. Thanks for feedback 👍

File: pages/linux/lslocks.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+> List local system locks.
+
+- List all local system locks:
+
+`lslocks`
+
+- List locks with defined column headers:
+
+`lslocks --output PID,COMMAND,PATH`
```
Comment (agnivade) [inline pages/linux/lslocks.md]: Should PID,COMMAND,PATH be within tokens (`{{}}`) ?

Comment (mebeim) [reply pages/linux/lslocks.md]: You're right, it should.

**Comment by RH-sdavey** [comment]
Changes made, thanks for merge, will add more updates/new pages later this week.

**Pull request merged**